fga-eps-mds/2020.1-stay-safe-user-service
# Change rating model

## Issue 
Resolves #37 


## Description
This PR changes the rating model to a more scalable one. Now the details is a composite type attribute that has the details inside. Now the possible details are lighting, movement_of_people and police_rounds.


## Checklist:



## How to validate?
Run the tests
Tests the rating crud manually and the get neighborhoods routes and check if the statistics are correct.
